homeassistant/components/renault/renault_hub.py

```diff
@@ -27,8 +27,14 @@
 if TYPE_CHECKING:
     from . import RenaultConfigEntry
 
-from .const import CONF_KAMEREON_ACCOUNT_ID, DEFAULT_SCAN_INTERVAL
-from .renault_vehicle import RenaultVehicleProxy
+from time import time
+
+from .const import (
+    CONF_KAMEREON_ACCOUNT_ID,
+    COOLING_UPDATES_SECONDS,
+    MAX_CALLS_PER_HOURS,
+)
+from .renault_vehicle import COORDINATORS, RenaultVehicleProxy
```

Comment:
Please keep MAX_CALLS_PER_HOURS for a follow-up PR

```suggestion
from .const import (
    CONF_KAMEREON_ACCOUNT_ID,
    COOLING_UPDATES_SECONDS,
    DEFAULT_SCAN_INTERVAL,
)
from .renault_vehicle import RenaultVehicleProxy
```

Reply:
done